commands/utils/params.go

```diff
@@ -507,58 +522,23 @@ func getBoolEnv(envKey string, defaultValue bool) (bool, error) {
-	// The repo name must be set as a part of the envs.
-	if gitParams.RepoName == "" {
-		return nil, &ErrMissingEnv{GitRepoEnv}
-	}
-	var project Project
-	if err := extractProjectParamsFromEnv(&project); err != nil {
-		return nil, err
-	}
-	params := Params{Git: *gitParams}
-	repo := FrogbotRepoConfig{Params: params, Server: *server}
-	if err := extractRepoParamsFromEnv(&repo); err != nil {
-		return nil, err
-	}
-	if err := extractGitNamingTemplatesFromEnv(&params.Git); err != nil {
-		return nil, err
-	}
-	repo.JfrogReleasesRepo = releasesRepo
-	repo.Projects = append(repo.Projects, project)
-	repo.OutputWriter = GetCompatibleOutputWriter(gitParams.GitProvider)
-	return FrogbotConfigAggregator{repo}, nil
-}
-
 // readConfigFromTarget reads the .frogbot/frogbot-config.yml from the target repository
-func readConfigFromTarget(client vcsclient.VcsClient) (configContent []byte, err error) {
-	e := &ErrMissingEnv{}
-	var branch string
-	if err = readParamFromEnv(GitBaseBranchEnv, &branch); err != nil && !e.IsMissingEnvErr(err) {
-		return nil, err
-	}
-	var repo string
-	if err = readParamFromEnv(GitRepoEnv, &repo); err != nil && !e.IsMissingEnvErr(err) {
-		return nil, err
-	}
-	var owner string
-	if err = readParamFromEnv(GitRepoOwnerEnv, &owner); err != nil && !e.IsMissingEnvErr(err) {
-		return nil, err
-	}
-
-	if repo != "" && owner != "" {
-		if branch == "" {
+func readConfigFromTarget(client vcsclient.VcsClient, clientInfo *ClientInfo) (configContent []byte, err error) {
+	if clientInfo.RepoName != "" && clientInfo.RepoOwner != "" {
+		log.Debug("Downloading", FrogbotConfigFile, "from target", clientInfo.RepoOwner, "/", clientInfo.RepoName)
+		var branch string
+		if len(clientInfo.Branches) == 0 {
 			log.Debug(GitBaseBranchEnv, "is missing. Assuming that the", FrogbotConfigFile, "file exists on default branch")
+		} else {
+			branch = clientInfo.Branches[0]
+			log.Debug("the", FrogbotConfigFile, "will be downloaded from the", branch, "branch")
 		}
 
-		log.Debug("Downloading", FrogbotConfigFile, "from target", owner, "/", repo, "/", branch)
 		gitFrogbotConfigPath := fmt.Sprintf("%s/%s", frogbotConfigDir, FrogbotConfigFile)
 		var statusCode int
-		configContent, statusCode, err = client.DownloadFileFromRepo(context.Background(), owner, repo, branch, gitFrogbotConfigPath)
+		configContent, statusCode, err = client.DownloadFileFromRepo(context.Background(), clientInfo.RepoOwner, clientInfo.RepoName, branch, gitFrogbotConfigPath)
 		if statusCode == http.StatusNotFound {
-			log.Debug(gitFrogbotConfigPath, "wasn't found on", owner, "/", repo)
+			log.Debug(fmt.Sprintf("the %s file wasn't recognized in the %s repository owned by %s", gitFrogbotConfigPath, clientInfo.RepoName, clientInfo.RepoOwner))
 			// If .frogbot/frogbot-config.yml isn't found, we'll try to run Frogbot using environment variables
 			return nil, &ErrMissingConfig{errFrogbotConfigNotFound.Error()}
 		}
```

Comment:
At the bottom of this function. the following - 
```
		if err != nil {
			return nil, err
		}
	}

	return configContent, nil
```
can be shortened to - 
```
	}

	return configContent, err
```

Reply:
done